pages/portfolio/create.tsx

```diff
@@ -26,6 +34,13 @@ const CreatePortfolioPage: NextPage = () => {
     const portfolio = await portfolioClient.put({
       content: {
         title,
+        name,
+        headshot,
+        about,
+        work,
+        skills,
+        projects,
+        contact,
```

Comment:
The build is failing because this is missing `message`. Try running `npm run build` before submitting a PR to check for errors

Reply:
Ah that makes sense, we decided to remove message and didn't do a full sweep to remove it completely, this will be fixed thank you.